ls1intum/Artemis
# `Development`: Add documentation for user experience

### Checklist
#### General



### Motivation and Context



Documentation for features supporting user satisfaction are currently missing.
See [the confluence page](https://confluence.ase.in.tum.de/display/ArTEMiS/Feature+documentation).

### Description


Add documentation pages for features that enhance the user experience:
- Guided Tours
- Theming

### Steps for Testing


- Proof read the new pages at https://artemis-platform--5368.org.readthedocs.build/en/5368/user/user-experience/
- Deploy the PR to a test server and check that the new link is present on the About Us page (but it will not work until merged, of course)

### Review Progress





#### Text Review
